pkg/monitortests/kubeapiserver/auditloganalyzer/audit_log_summarizer.go

```diff
@@ -468,8 +490,14 @@ func writeAuditLogDL(artifactDir, timeSuffix string, auditLogSummary *AuditLogSu
 		for rk, rv := range uv.perResourceRequestCount {
 			for vk, vv := range rv.perVerbRequestCount {
 				for sk, sv := range vv.perHTTPStatusRequestCount {
-					rows = append(rows, map[string]string{"User": uv.user, "Resource": rk.Resource, "Verb": vk, "HttpStatus": strconv.FormatInt(int64(sk), 10), "RequestCount": strconv.FormatInt(int64(sv), 10)})
+					user, unmodifiedUser := cleanupUser(uv.user)
```

Comment:
does this need to have a single record for all of the same name?

Reply:
I [map](https://github.com/openshift/origin/blob/8bced75d5a89bb5836fd804b891a50a61fcc9c94/pkg/monitortests/kubeapiserver/auditloganalyzer/audit_log_summarizer.go#L477) the unique user value to a standarized prefix-count value but maintain the records separately so we don't have to sum them all up and use only a single record.  Could if we needed too but seemed more straightforward to keep them separate.